### PR TITLE
[MOS-392] Increase the stack size for tasks

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
**Description**

Increased stack for:
ApplicationSpecialInput, FileIndexer and ApplicationManager

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [x] Has documentation updated

### CODE REVIEW

File: module-apps/application-special-input/ApplicationSpecialInput.cpp [hunk 1]
```diff
@@ -9,7 +9,7 @@ using namespace app;
 
 namespace
 {
-    constexpr auto SpecialInputAppStackDepth = 2200;
+    constexpr auto SpecialInputAppStackDepth = 512 * 5;
```
Comment (pholat) [inline module-apps/application-special-input/ApplicationSpecialInput.cpp:12]: tbh, why not full number? :)

File: module-services/service-fileindexer/ServiceFileIndexer.cpp [hunk 2]
```diff
@@ -14,13 +14,14 @@ namespace
     {
         return purefs::createPath(purefs::dir::getUserDiskPath(), "music").string();
     }
+    inline constexpr auto fileIndexerServiceStackSize = 1024 * 5;
```
Comment (pholat) [inline module-services/service-fileindexer/ServiceFileIndexer.cpp:17]: it's actually a pretty big bump, what happened?

Comment (Maciej-Mudita) [reply module-services/service-fileindexer/ServiceFileIndexer.cpp:17]: Currently, only 7% of the space remains. We need more stock - I set it to at least 20%.
stackstats

File: module-apps/application-special-input/ApplicationSpecialInput.cpp (hunk 1 above)
Comment (Maciej-Mudita) [reply module-apps/application-special-input/ApplicationSpecialInput.cpp:12]: I think it is clearer, but that's just my opinion.

**Review by pholat — APPROVED** [review]

**Review by Lefucjusz — APPROVED** [review]

**Pull request merged**